pkg/config/config_template.yaml

```diff
@@ -106,14 +106,30 @@ api_key:
 ## @env DD_TAGS - space separated list of strings - optional
 ## List of host tags. Attached in-app to every metric, event, log, trace, and service check emitted by this Agent.
 ##
-## Additional tags can be supplied using the `DD_EXTRA_TAGS` environment variable.
+## This configuration value is merged with DD_EXTRA_TAGS.  This allows some
+## tags to be set in a configuration file (tags), with additional tags added
```

Comment:
Trying to make it clearer that this instance of `tags` refers to the config option name:

```suggestion
## tags to be set in a configuration file (`tags`), with additional tags added
```